src/raftstore/store/worker/apply.rs

```diff
@@ -843,9 +843,6 @@ impl ApplyDelegate {
 
         let split_key = split_req.get_split_key();
         let mut region = self.region.clone();
-        if split_key <= region.get_start_key() {
```

Comment:
the split key can be equal to the start key, so we can't ignore this check here.